src/Microsoft.AspNetCore.BuildTools.CreateLZMA/CreateLZMA.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using Microsoft.Build.Framework;
+using Microsoft.DotNet.Archive;
+using MSBuildTask = Microsoft.Build.Utilities.Task;
+
+namespace Microsoft.AspNetCore.BuildTools.LZMA
+{
+    public class CreateLZMA : MSBuildTask
+    {
+        [Required]
+        public string OutputArchiveFilePath { get; set; }
```

Comment:
Can we make the API for this task align with the patterns we've established for other tasks? We did an API review and came up with this for zips: https://github.com/dotnet/arcade/tree/master/src/Microsoft.DotNet.Build.Tasks.IO#ziparchive. (Snippet copied below)

-----------

### `ZipArchive`

Creates a .zip archive file

#### Common parameters
Task parameter     | Type    | Description
-------------------|---------|---------------------------------------------------------------
OutputPath         | string  | **[Required]**  The path where the zip file should be created. The containing directory will be created if it doesn't already exist.
Overwrite          | boolean | Overwrite output path if it exists

There are two valid usages: a list of files, or an entire directory

#### Parameter set - files

Task parameter     | Type        | Description
-------------------|-------------|----------------------------------------------------------------------------------------------------------------------------------
SourceFiles        | ITaskItem[] | **[Required]** Files to be included in the zip. <br> The `Link` metadata item can be set to explicitly set the zip entry path.
BaseDirectory      | string      | **[Required]** The directory to use as the base directory. The entry path for each item in SourceFiles is relative to this.

```xml
<ItemGroup>
   <JsFiles Include="$(PublishDir)**\*.js" >
   <JsFiles Include="../webpack.js" Link="webpack.js" >
</ItemGroup>

<ZipArchive OutputPath="myapp.js.zip" SourceFiles="@(JsFiles)" BaseDirectory="$(PublishDir)" />
```


#### Parameter set - directory

Task parameter     | Type    | Description
-------------------|---------|-------------
SourceDirectory    | string  | **[Required]** Creates a zip for an entire directory.

Example:
```xml
<ZipArchive OutputPath="myapp.zip" SourceDirectory="$(PublishDir)" />
```